.github/workflows/ci.yml

```diff
@@ -133,6 +133,16 @@ jobs:
         run: ci/valgrind-check/run.sh
         shell: bash
 
+  typo-check:
+    name: Typo Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this repository 
+        uses: actions/checkout@v4
+
+      - name: Check spelling
+        uses: crate-ci/typos@master
+
```

Comment:
@kilpkonn thanks for the contribution! Could you add `typo-check` to the `needs` list in the "CI status check" below?

Reply:
Will do!